html_css/intermediate_css/custom-properties.md

```diff
@@ -139,7 +139,7 @@ Using the `prefers-color-scheme` media query can be pretty helpful for users sin
 
 ### Additional Resources
 
-This section contains helpful links to other content. It isn’t required, so consider it supplemental for if you need to dive deeper into something.
+1. [This video](https://www.youtube.com/watch?v=PHO6TBq_auI) is a great introduction to CSS custom properties. Go ahead and watch it.
```

Comment:
One minor change regarding link text. We should make the link text more descriptive here:

```suggestion
1. This [video on CSS custom properties](https://www.youtube.com/watch?v=PHO6TBq_auI) is a great introduction. Go ahead and watch it.
```